hashintel/hash
# H-866: Implement pinned entity types on User and Org profile pages

## 🌟 What is the purpose of this PR?




This PR adds the ability to pin entity types to a user's profile page.

In addition, it addressed the outstanding review comments left in https://github.com/hashintel/hash/pull/3319.

Remaining todos:

- [x] pluralise entity type tab titles
- [x] use link icon in edit pinned entity types modal

## 🔗 Related links





- H-866

## 🔍 What does this change?




See commits.

## Pre-Merge Checklist 🚀

### 🚢 Has this modified a publishable library?




This PR:


### 📜 Does this require a change to the docs?




The changes in this PR:


### 🕸️ Does this require a change to the Turbo Graph?




The changes in this PR:


## ⚠️ Known issues



## 🐾 Next steps



## 🛡 What tests cover this?



None

## ❓ How to test this?



❗️ This PR cannot be tested in the Vercel deployment because it requires backend changes to be applied

1. Go to your user profile page
2. Toggle the tabs, there should be "Profile" and "Pages" initially
3. Add additional pinned entity type tabs by clicking "Modify"

## 📹 Demo



https://github.com/hashintel/hash/assets/42802102/c25eefab-757f-485e-9cae-693d3efa6ccc
